packages/video_player/video_player/ios/Classes/FLTVideoPlayerPlugin.m

```diff
@@ -369,6 +375,10 @@ - (void)setIsLooping:(bool)isLooping {
   _isLooping = isLooping;
 }
 
+- (void)setInternalPlaybackSpeed:(double)speed {
```

Comment:
Why are you overriding a property setter with the default implementation of a property setter.

Reply:
I was copying the pattern I saw above, but after referring to [Apple docs](https://developer.apple.com/library/archive/documentation/Cocoa/Conceptual/ObjectiveC/Chapters/ocProperties.html), I see that property declarations give us the getter/setters for free. Nice! Will remove